src/libsync/theme.cpp

```diff
@@ -374,6 +374,10 @@ QString Theme::aboutVersions(Theme::VersionFormat format) const
             gitUrl = gitSHA1(format) + br;
         }
     }
+    QStringList sysInfo = {QStringLiteral("OS: %1-%2").arg(QSysInfo::productType(), QSysInfo::kernelVersion())};
+    if (auto guiApp = qobject_cast<QGuiApplication *>(qApp)) {
```

Comment:
Maybe we should `OC_EXPECT` this cast. I can't think of a situation in which this would fail regularly. 

If this is called by the CLI, too, then we should add a comment that mentions this so people don't have to speculate.

Reply:
```suggestion
    // may be called by both GUI and CLI, but we can display QPA only for the former
    if (auto guiApp = qobject_cast<QGuiApplication *>(qApp)) {
```